src/mame/konami/hornet.cpp

```diff
@@ -588,6 +650,11 @@ uint8_t hornet_state::sysreg_r(offs_t offset)
 			    0x04 = ADEOC (ADC EOC)
 			    0x02 = ADDOR (ADC DOR)
 			    0x01 = ADDO (ADC DO)
+			
+			FIXME: Only gradius4 and nbapbp take advantage of JVS; the rest do not.
+			In fact, the rest of the games might NOT like it if a JVS I/O board is
+			plugged in (or the SENSE line is enabled). This needs verifcation on
+			actual hardware.
```

Comment:
If that's the case, I may as well remove this note. It's interesting Konami would leave all this leftover behavior unknown to all the cab operators or owners way back in the day when it was new.

I'm now curious if the nwk-tr games have master calendar functionality because nwk-tr boards have a JVS I/O port too.

Reply:
Not exactly relevant to this PR specifically but yes, all of the games in the nwk-tr driver also support the Master Calendar JVS device.